src/ingest/__init__.py

```diff
@@ -0,0 +1,169 @@
+# TODO : setup rabbit listeners; "update_es": [informationmodels,concepts,datasets,dataservices]
+import json
+import math
+import time
+
+import requests
+import os
+from elasticsearch import Elasticsearch, helpers
+from .utils import Indexes
+from jsonpath_ng import parse
+
+ES_HOST = os.getenv('ELASTIC_HOST') or "localhost"
+ES_PORT = os.getenv('ELASTIC_PORT') or "9200"
+client = Elasticsearch([ES_HOST + ':' + ES_PORT])
+API_URL = os.getenv('API_URL')
+
+
+def fetch_from_services():
+    start = time.time()
+    info_status = fetch_information_models()
+    concept_status = fetch_concepts()
+    service_status = fetch_dataservices()
+    datasets_status = fetch_datasets()
+    total_time = time.time() - start
+    total_concepts = 0
+    for iteration in concept_status:
+        total_concepts = total_concepts + iteration[0]
+    totalElements = info_status[0]+total_concepts+service_status[0]+datasets_status[0]
+    result = {
+        "took": total_time,
+        "total": totalElements,
+        "informationmodels": info_status,
+        "concepts": concept_status,
+        "dataservice": service_status,
+        "datasets": datasets_status
+    }
+    return result
+
+
+def fetch_information_models():
+    print("fetching information models")
+    info_url = API_URL + "informationmodels"
+    size = requests.request(url=info_url, method="GET")
```

Comment:
Default timeout på `requests` sine kall er None hvis jeg husker riktig. Så i verste fall vil denne linjen vente evig på svar hvis ressursen er nede. Legg til en timeout for å unngå det.

Legg også til exception handling for både rest exceptions og status != 2xx or 3xx 

```suggestion
from requests.exceptions import RequestException, HTTPError
    try:
        r = requests.get(url=info_url, timeout=5)
        r.raise_for_status() # turns status 4xx and 5xx into exceptions
    except HTTPError as e:
        # handle 4xx and 5xx
        pass
    except RequestException as e:
        # Timeout, TooManyRedirects, ConnectionRefused etc.
        pass
```